geospaas_rest_api/tests.py

```diff
@@ -1,3 +1,146 @@
 from django.test import TestCase
+from django.test import Client
+from geospaas.vocabularies.models import Platform, Instrument 
+from geospaas.vocabularies.models import Parameter, DataCenter
+import json
 
 # Create your tests here.
+class DatasetURITests2(TestCase):
+
+
+    fixtures = ["vocabularies", "catalog"]
+    def setUp(self):
+        self.c = Client()
+
+    def test_api_initial_calls(self):
+        
+        response = self.c.get('//api/')
+        self.assertEqual(response.status_code, 200)
+
+
+    def test_api_initial_calls2(self):
+
+        response2 = self.c.get('//api/geographic_locations/')   
+        self.assertEqual(response2.status_code, 200)
+        self.assertContains(response2, 'geometry')
```

Comment:
As I wrote before it is not sufficient to use assertContains.
Use assertEqual as in my example above.
Here and in all other tests.